### ISSUE TITLE
swipe back

### ISSUE THREAD

**Issue by keenan35i** [Telerik-Verified-Plugins/NativePageTransitions#143]
Is this plugin eventually going to support swipe to prev page? thanks!

**Comment by EddyVerbruggen** [comment]
No, as that's something you can wire up from JS on a per-usage basis..

**Comment by keenan35i** [comment]
yes it can be done with js but using the prev screenshot on the back would make for much better scalability with large amounts of content

**Comment by EddyVerbruggen** [comment]
I understand. Reopening just to keep it on the radar.

**Comment by keenan35i** [comment]
alright thanks!

**Comment by kations** [comment]
Best would be to use the prev Screenshot on drag, and when i drag far enough and release, the back animation completes and switch to the web-view after the animation ended with a small delay. So the web-view has some time to render the prev template and restore scroll position etc.

Would be nice if someone with the skills will have time to make this happen

**Comment by vtntimo** [comment]
I sent email to you guys regarding this. Please check :)

**Comment by keenan35i** [comment]
Any update on this? Would be great to have this feature.

**Comment by AsimNet** [comment]
how to disable swipe back feature in iOS?
it's buggy
